test/e2e/nstemplatetier_test.go

```diff
@@ -100,7 +101,7 @@ func TestUpdateNSTemplateTier(t *testing.T) {
 
 	count := 2*MaxPoolSize + 1
 	ctx, awaitility := testsupport.WaitForDeployments(t, &toolchainv1alpha1.NSTemplateTier{})
-	defer ctx.Cleanup()
+	// defer ctx.Cleanup()
```

Comment:
Why not performing cleanup?

Reply:
it's a left-over: at some point I commented out this line so I could see how the resources looked like at the end of the test. This was not intended to be part of the final PR 🤦‍♂️

Reply:
reverted in a21cbb0